dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Basics/SineWave.mo

```diff
@@ -0,0 +1,30 @@
+within Dynawo.Electrical.Controls.Basics;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools
+* for power systems.
+*/
+
+model SineWave "Parameterizable sine wave model"
+  import Dynawo.Connectors;
+
+  Connectors.ImPin source(value(start = Value0));
+
+  parameter Real K "Amplitude of the sine wave";
+  parameter Types.AngularVelocity Omega "Pulsation of the sine wave";
```

Comment:
import Types missing, no ?

Reply:
In the encapsulated package Dynawo, there is 

import Dynawo.Types;

So this line is not needed in the package.